CONTRIBUTING.md

```diff
@@ -1,3 +1,12 @@
+[Contributing](#contributing)  
+[Resolving Merge Conflicts](#resolving-merge-conflicts)
+[Unity Version](#unity-version)  
```

Comment:
needs 'new line' as "unity version" gets placed on the same line as "resolving merge.."